### DISCUSSION TITLE
Yank/move to new file?

### DISCUSSION THREAD

**Discussion by tobarja** [Zettlr/Zettlr#4490] [Ideas]
I started moving data from a very long Notepad++ document into Zettlr. After pasting, the very next thing I wanted to do was select some text and move it into a brand new document. 

ie: My document has text from many random things in it. I want to quickly migrate that text into many new documents (one at a time, of course), without having to cut, new file, paste.

Anyone else?

**Comment by nathanlesage** [comment]
There has been a request for this feature from quite some time ago, but I haven't yet figured a good way to include that yet 😶

**Comment by exanrauzer** [comment]
This could be a very handy feature. You can select a portion of text, open the context menu, and choose "move to a new file." The file will be created, a new tab will open, and the content will be pasted. Nice and powerful.
